src/components/fields/SchemaField.js

```diff
@@ -156,6 +160,7 @@ function SchemaField(props) {
   return (
     <Wrapper
       label={props.schema.title || schema.title || name}
+      description={props.schema.description}
```

Comment:
We should rather retrieve this from the resolved schema, so `schema.description` here.